test/integration/framework/etcd.go

```diff
@@ -129,8 +129,17 @@ func RunCustomEtcd(dataDir string, customFlags []string) (url string, stopFn fun
 	cmd.Stderr = os.Stderr
 	stop := func() {
 		cancel()
-		err := cmd.Wait()
-		klog.Infof("etcd exit status: %v", err)
+		errCh := make(chan error, 1)
```

Comment:
Is this where we're hanging?

Reply:
This is the stop function called when a package is done running tests, right? I thought we were hanging in the middle of particular test functions

Reply:
hmm, so it is possible that we never get here and we get stuck in one test?

Reply:
don-t we run the test with a `timeout` flag?

Reply:
duh, this is the RunCustomEtcd, the tests on the jobs run the external etcd